airflow/utils/sqlalchemy.py

```diff
@@ -39,7 +39,7 @@ def setup_event_handlers(engine,
                          initial_backoff_seconds=0.2,
                          max_backoff_seconds=120):
     @event.listens_for(engine, "engine_connect")
-    def ping_connection(connection, branch):
+    def ping_connection(connection, branch): # pylint: disable=unused-variable
```

Comment:
Same here, for example. I am just curious what variable that should be?

Reply:
```
airflow/utils/sqlalchemy.py:42:4: W0612: Unused variable 'ping_connection' (unused-variable)
```